src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,68 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DeadlineTask extends Task {
+
+    LocalDate date;
+
+    public DeadlineTask(String input) {
+        super(input);
+        this.type = "deadline";
+        this.date = this.getDate();
+        this.updateDescription();
+    }
+
+    /**
+     * Format DeadlineTask description into display format for Duke
+     */
+    private void updateDescription() {
+        this.description += this.getInfo();
+        this.description += "(by: ";
+        this.description += this.date.format(DateTimeFormatter.ofPattern("d MMM yyyy"));
+        this.description += ")";
```

Comment:
I like your proper use of method naming standards(descriptive, camelCased verbs)